lsp-mode.el

```diff
@@ -1059,14 +1059,61 @@ every element of it is of type string, else nil."
    (vectorp candidate)
    (seq-every-p #'stringp candidate)))
 
+(make-obsolete 'lsp--string-vector-p nil "7.0.2")
```

Comment:
The version should be `7.1` now

Reply:
Why? 

@ericdallo as a side note, I think that we should bump the version in lsp-mode.el after the release, not before it to avoid that kind of discussions.

Reply:
Why as in?

Reply:
(my why was for @kiennq )

Reply:
I saw 7.0.1 has been release, and `lsp-modeline` has adopt 7.1 already

Reply:
So I should change to 7.1?

Reply:
Alternatively, I think you can define a `alpha` version for anything that's in development, then change all of them to exact version number when release. Contributor can know exact version (`alpha`) then

Reply:
Who would change the alpha to a concrete version?

Reply:
The one who release new version? Mostly @yyoncho and @ericdallo. Using `alpha` is just my proposal to they though

Reply:
I've changed to 7.1. Please update it if necessary before release.